spec/unit/shoulda/matchers/active_record/have_rich_text_matcher_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'unit_spec_helper'
+
+describe Shoulda::Matchers::ActiveRecord::HaveRichText, type: :model do
+
+  if active_record_supports_has_rich_text?
```

Comment:
It would be nice to have tests for falsy `active_record_supports_has_rich_text?`.

Reply:
Sure! Before this branch is merged I'll added that!

Reply:
Done!

Please, take a look into the if conditions. The difference between the second and the third is the name of the class inside the message.  

In Rails 4 the test returns a NillClass, but in Rails 5 or greater the test return a String class.

This is what I got when I run the tests with Rails 4: 

``` 
activerecord-4.2.11.1/lib/active_record/connection_adapters/abstract_adapter.rb:84: warning: deprecated Object#=~ is called on Integer; it always returns nil
```

I'll create some `shared examples` for the duplications.